packages/go_router/lib/src/route.dart

```diff
@@ -279,3 +317,62 @@ class GoRoute {
   ) =>
       const SizedBox.shrink();
 }
+
+/// A route that displays a UI shell around the matching child route. Builds
+/// a new Navigator that is used to display any matching sub-routes, instead
+/// of placing them on the root Navigator.
+///
+/// To display a child route on a different Navigator, provide it with a
```

Comment:
maybe add some sample code for this. This should probably be mentioned after the example below

Reply:
9378fd5ace20f39d65c053e2891afd072a5168ac